packages/schemas/src/types/logto-config/index.ts

```diff
@@ -45,6 +51,55 @@ export const logtoOidcConfigGuard: Readonly<{
   [LogtoOidcConfigKey.CookieKeys]: oidcConfigKeyGuard.array(),
 });
 
+/**
+ * Logto JWT customizer token types, used in REST API routes.
+ */
+export enum LogtoJwtTokenKeyType {
+  AccessToken = 'access-token',
+  ClientCredentials = 'client-credentials',
+}
+
+export enum LogtoJwtTokenKey {
+  AccessToken = 'jwt.accessToken',
+  ClientCredentials = 'jwt.clientCredentials',
+}
```

Comment:
what's the difference between these two?

Reply:
The `LogtoJwtTokenKeyType` is used to guard the API path and `LogtoJwtTokenKey` is used to guard the key in the `logto_config` table. Can refer to `LogtoOidcConfigKeyType` and `LogtoOidcConfigKey` in this file.